packages/modules/src/modules/banners/choiceCardsButtonsBanner/choiceCardsButtonsBannerStyles.ts

```diff
@@ -60,6 +60,16 @@ export const columnMarginOverrides = css`
     }
 `;
 
+export const eol_choiceCardverticlaAlignment = css`
+    ${from.tablet} {
+        justify-content: flex-start;
+        margin-top: 6.6rem;
+    }
+    ${from.desktop} {
+        margin-top: 7.6rem;
+    }
+`;
+
```

Comment:
I've added this css in so that the ChoiceCardsButtonsBanner component banner renders similarly to the way it did before with an arbitrary margin top in order to make it align with the left hand side copy. The templated banners which will replace this component now use grid and that necessitated removing the original margin top values in order that the placement can be properly manage by the grid columns and rows.

At the earliest opportunity, this component (ChoiceCardsButtonsBanner) should be replaced with the new template banner.